src/patternfly/components/Page/page.scss

```diff
@@ -155,22 +171,39 @@
   --pf-c-page__main-section--m-dark-100--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-100);
   --pf-c-page__main-section--m-dark-200--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-200);
 
+  // Drawer
+  --pf-c-page__drawer--ZIndex: var(--pf-global--ZIndex--xs);
```

Comment:
Are you going to add a transition like the sidebar has?

Reply:
Nope, but we can. What do you think @maryshak1996 @mceledonia?

Reply:
@mcoker I would think that the same transition used with the vertical nav is what would make sense here

Reply:
@christiemolloy added the transition if you want to have another look.

Reply:
Transition looks really good!!

Reply:
Can you show an example of the sidebar expanded and the notification drawer expanded @mcoker